bin/import_events.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import argparse
+import csv
+import json
+import requests
+
+
+def import_events(url, csv_path, dry_run):
+    events = parse_csv(csv_path)
+    print("Found {num} records".format(num=len(events)))
+    return post_events(url, events, dry_run=dry_run)
+
+
+def parse_csv(csv_path):
+
+    with open(csv_path, mode='r') as csv_file:
+        reader = csv.DictReader(csv_file)
+        events = []
+        for row in reader:
+            events.append(create_event(row['event_type'], row['account'], row['instance'], row['timestamp'],
```

Comment:
Maybe just create an event object?  Similar to example 1 in https://stackoverflow.com/a/2466380/6124862

Reply:
DONE - Thanks!

Reply:
Forgot to clean  up...

Reply:
Now it is done! :)